packages/terra-menu/tests/wdio/menu-spec.js

```diff
@@ -0,0 +1,157 @@
+      browser.click('#large-menu-button');
+    });
+
+    Terra.should.matchScreenshot({ selector: '#root' });
+    Terra.should.beAccessible();
+  });
+
+  describe('Menu-Non-Selectable', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-menu/menu/menu/non-selectable-menu');
+      browser.click('#non-selectable-menu-button');
+    });
+
+    Terra.should.matchScreenshot({ selector: '#root' });
+    Terra.should.beAccessible();
+  });
+
+  describe('Menu-Selectable', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-menu/menu/menu/selectable-menu');
+      browser.click('#selectable-menu-button');
+      browser.click('.TestGroupItem3');
+    });
+
+    Terra.should.matchScreenshot({ selector: '#root' });
+    Terra.should.beAccessible();
+  });
+
+  describe('Menu-Selectable with Varying Items', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-menu/menu/menu/selectable-and-unselectable-items-menu');
+      browser.click('#default-button');
+    });
+
+    Terra.should.matchScreenshot({ selector: '#root' });
+    Terra.should.beAccessible();
+
+    it('selects an item and maintains selection after menu has been reopened', () => {
+      browser.click('.TestSelectableItem');
+      browser.click('#default-button');
+    });
+    Terra.should.matchScreenshot('maintained selection after reopen', { selector: '#root' });
+  });
+
+  describe('Menu with a submenu', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-menu/menu/menu/sub-menu');
+      browser.click('#sub-menu-button');
+    });
+
+    Terra.should.matchScreenshot('main menu', { selector: '#root' });
+    Terra.should.beAccessible();
+
+    it('displays the submenu', () => {
+      browser.click('.TestNestedMenu');
+    });
+    Terra.should.matchScreenshot('submenu', { selector: '#root' });
+  });
+
+  describe('Menu Keyboard Navigation-Arrow Keys', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-menu/menu/menu/sub-menu');
+      browser.click('#sub-menu-button');
+    });
+
+    it('does not do anything when left arrow is pressed on the first layer', () => {
+      browser.keys(['Tab', 'ArrowLeft']);
+    });
+    Terra.should.matchScreenshot('main menu remains open', { selector: '#root' });
+
+    it('displays the submenu on right arrow', () => {
+      browser.keys(['Tab', 'Tab', 'ArrowRight']);
+    });
+    Terra.should.matchScreenshot('navigated to submenu', { selector: '#root' });
+
+    it('returns to the main menu on left arrow', () => {
+      browser.keys(['Tab', 'Tab', 'ArrowLeft']);
+    });
+    Terra.should.matchScreenshot('returned to main menu', { selector: '#root' });
+  });
+
+  describe('Menu Keyboard Navigation-Enter Key', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-menu/menu/menu/sub-menu');
+      browser.click('#sub-menu-button');
+    });
+
+    it('displays the submenu on right arrow', () => {
```

Comment:
I think you meant to name the test '_displays the submenu on Enter keypress_'

and the one on line 152 '_returns to the main menu on Enter keypress_'

Reply:
Yeah lets get these test names updated so the screenshots match before we merge.